misc/start_faux

```diff
@@ -64,12 +64,6 @@ if [ -n "${options[telnet]}" ]; then
     (while true; do echo Telnet `hostname`; nc -nvlt -p 23 -e `which hostname`; done) &
 fi
 
-# Setup dummy https listener to trigger port-scan failure
```

Comment:
Is this mock https server important ??

Reply:
Nope -- as long as "443" is represented somehow it'll be fine. I think it's only used for nmap scanning or something.

Reply:
so if you use the system.conf option startup_faux_opts=tls then this will run else not.
Would you prefer the argument remain https ?

Reply:
the https was for show only (opening the port), so whatever works is fine. Yous is strictly better as long as the tests pass!